deployment/src/test/resources/application-extension-loading-enabled.properties

```diff
@@ -0,0 +1,2 @@
+quarkus.wiremock.devservices.global-response-templating=true
+quarkus.wiremock.devservices.extension-scanning-enabled=true
```

Comment:
Would you be so kind to include the new config parameter in the documentation as well - thank you very much
`docs/modules/ROOT/pages/includes/quarkus-wiremock.adoc`

Reply:
Isn't this meant to be updated automatically when building the project? Any idea why it wasn't updated?

Reply:
Good point. I'm aware that this works when contributing to the platform. That's why the config comments are mandatory, right? However, I haven't thought about how this should work for extensions. I've just maintained them manually. Let me check ... 😄

Reply:
I might be mistaken, but I am pretty sure it work automatically. @gastaldi can you confirm?

Reply:
I believe this needs to be changed to match the config document: https://github.com/quarkiverse/quarkus-wiremock/blob/main/docs/pom.xml#L68

Reply:
@gastaldi Ah damn, thanks! I'll change it.

Reply:
Thanks @gastaldi!